OpenRA.Game/Selection.cs

```diff
@@ -59,7 +59,11 @@ public void Combine(World world, IEnumerable<Actor> newSelection, bool isCombine
 
 			var voicedActor = actors.FirstOrDefault(a => a.Owner == world.LocalPlayer && a.IsInWorld && a.HasTrait<IVoiced>());
 			if (voicedActor != null)
-				voicedActor.PlayVoice("Select");
+			{
+				var selectable = voicedActor.TraitOrDefault<Selectable>();
+				if (selectable != null)
```

Comment:
This should be merged into the `voicedActor` query to ensure that the selected actor will always play a voice.